drivers/hmis/app/models/hmis/user.rb

```diff
@@ -60,12 +60,17 @@ def load_effective_permissions
     define_method("#{permission}_for?") do |entity|
       return false unless send("#{permission}?")
 
+      # Use global permission scope if client is unenrolled
+      return send("#{permission}?") if entity.is_a?(Hmis::Hud::Client) && entity.projects.empty?
+
       base_entities = permissions_base_for_entity(entity)
 
       # No entity was specified and this permission is allowed to be global (for example Client access)
-      return true if !base_entities.present? && ::Hmis::Role.global_permissions.include?(permission)
+      return true if base_entities.nil? && ::Hmis::Role.global_permissions.include?(permission)
+
+      raise "Invalid entity '#{entity.class.name}' for permission '#{permission}'" if base_entities.nil?
 
-      raise "Invalid entity '#{entity.class.name}' for permission '#{permission}'" unless base_entities.present?
+      return false unless base_entities.present?
```

Comment:
Is it possible for base_entities to be empty but not nil?  I assume that'd be an empty array.  It seems odd that nil and [] would have different results, maybe a comment as to why would be appropriate?